src/EnergyPlus/CommandLineInterface.hh

```diff
@@ -62,6 +62,12 @@ struct EnergyPlusData;
 
 namespace CommandLineInterface {
 
+    enum class ReturnCodes {
+        Success,
+        Failure,
+        SuccessButHelper,
+    };
```

Comment:
Return codes

Reply:
It looks like these are being used as raw integers everywhere.  Is it worth having an enum class in this case?  I don't much like having lists of constant ints, but if they are _always_ statically cast to integers then it may be a cleaner approach.  And we could `constexpr int` them so they are _actually_ hanging out in the global state :sunglasses: 

What do you think?